packages/modules/payment/src/migrations/Migration20250123122334.ts

```diff
@@ -0,0 +1,27 @@
+import { Migration } from '@mikro-orm/migrations';
+
+export class Migration20250123122334 extends Migration {
+
+  override async up(): Promise<void> {
+    this.addSql(`create table if not exists "account_holder" ("id" text not null, "provider_id" text not null, "external_id" text not null, "email" text null, "data" jsonb not null default '{}', "metadata" jsonb null, "created_at" timestamptz not null default now(), "updated_at" timestamptz not null default now(), "deleted_at" timestamptz null, constraint "account_holder_pkey" primary key ("id"));`);
+    this.addSql(`CREATE INDEX IF NOT EXISTS "IDX_account_holder_deleted_at" ON "account_holder" (deleted_at) WHERE deleted_at IS NULL;`);
+    this.addSql(`CREATE UNIQUE INDEX IF NOT EXISTS "IDX_account_holder_provider_id_external_id_unique" ON "account_holder" (provider_id, external_id) WHERE deleted_at IS NULL;`);
+
+    this.addSql(`alter table if exists "payment_collection_payment_providers" drop constraint if exists "payment_collection_payment_providers_payment_coll_aa276_foreign";`);
```

Comment:
Not sure where these came from, any ideas if they are valid or not?

Reply:
They are valid but for some reason our migration generator wants to delete them and create them anew, as seen on line 11-14

Reply:
We can probably keep the migration as is to correct the casing convention (or whatever it is that triggers the recreation), so we don't keep seeing it moving forward.